src/components/comment/CommentForm.jsx

```diff
@@ -38,43 +33,46 @@ const CommentForm = (props) => {
     e.stopPropagation();
   };
 
-    const autoResizeTextArea = () => {
-        const textArea = formInputRef.current;
-        textArea.style.height = "auto";
-        let scrollHeight = textArea.scrollHeight;
-        textArea.style.height = `${scrollHeight}px`;
-        if (parseInt(textArea.style.height) > parseInt(MAX_TEXT_AREA_HEIGHT)) {
-            textArea.style.height = MAX_TEXT_AREA_HEIGHT
-        }
+  const autoResizeTextArea = () => {
+    const textArea = formInputRef.current;
+    textArea.style.height = 'auto';
+    let scrollHeight = textArea.scrollHeight;
+    textArea.style.height = `${scrollHeight}px`;
+    if (parseInt(textArea.style.height) > parseInt(MAX_TEXT_AREA_HEIGHT)) {
+      textArea.style.height = MAX_TEXT_AREA_HEIGHT;
     }
+  };
 
-    return (
-        <Form onSubmit={onSubmitHandler} onKeyUp={onKeyUpHandler} onClick={onClickHandler}>
-            <Form.Group className="m-2" controlId="formBasicComment">
-                <Col className="col-lg-12 p-0">
-                    <Row className="container-fluid p-0 m-0">
-                        <div id="comment-form">
-                            <Form.Control
-                                className="comment-form-control"
-                                name="comment"
-                                as="textarea"
-                                placeholder={placeholder}
-                                required
-                                value={commentValue}
-                                onChange={onValueChange}
-                                ref={formInputRef}
-                                onKeyPress={autoResizeTextArea}
-                                onKeyDown={autoResizeTextArea}
-                            />
-                            <Button className="comment-form-button" variant="primary" type="submit" >
-                                <ArrowRight/>
-                            </Button>
-                        </div>
-                    </Row>
-                </Col>
-            </Form.Group>
-        </Form>
-    );
+  return (
+    <Form onSubmit={onSubmitHandler} onKeyUp={onKeyUpHandler} onClick={onClickHandler}>
+      <Form.Group className="m-2" controlId="formBasicComment">
+        <Col className="col-lg-12 p-0">
+          <Row className="container-fluid p-0 m-0">
+            <div id="comment-form">
+              <Form.Control
+                className="comment-form-control"
+                name="comment"
+                as="textarea"
+                placeholder={intl.formatMessage({
+                  id: 'comment.form.placeholder',
+                  defaultMessage: 'Write your comments here  (Ctrl+Enter to confirm, Esc to cancel)'
```

Comment:
@LaChope 

What is the purpose of the defaultMessage? Isn't it useful to not write the same string twice? Whouldn't we say that "en" variant is the default here without need to specify "defaultMessage" ?

Reply:
@blcham It is just to cover the case where the option "locale" would be an empty string. Is it useful scenario to take into account or should we dismiss it? If we dismiss it defaultMessage is not necessary

Reply:
Thus I would assume we always have at least one "locale" complete -- we can consider 'en' locale for that case.